.github/workflows/build-docker.yml

```diff
@@ -34,3 +34,4 @@ jobs:
           tags: ${{ env.docker_username }}/python-bindings:${{ env.branch }},${{ env.docker_username }}/python-bindings:latest
           build-args: |
             branch=${{ env.branch }}
+            from=benjaminrodenberg/precice:develop
```

Comment:
We need to provide a docker image with the latest version of preCICE, because we need to include the breaking changes.